modules/ROOT/pages/tokenization.adoc

```diff
@@ -44,6 +44,22 @@ The Tokenization service provides many benefits:
 ** High-performance operation ensures low-latency processing.
 ** Performance scales linearly on each node with vertical and horizontal scaling for vaultless tokenization provided by the Anypoint Platform.
 
+== Prerequisites
+
+To configure and use the tokenization service, you must:
+
+* Have the Anypoint Security - Edge entitlement for your Anypoint Platform account. If you don't see *Security* in *Management Center* or the *Tokenization Service* tab in Runtime Manager, contact your customer success manager to enable the tokenization service for your account. +
```

Comment:
Delete this sentence "Have the Anypoint Security - Edge entitlement for your Anypoint Platform account."

The next sentence is incorrect also. You are trying to see if they have the entitlement. The next sentence should be something like "If you don't see Management Center->Security->Tokenization, contact your......

In addition the Tokenization permission must be added at Management Center->Permissions->Tokenization.  After adding these permissions, you should now see both Management Center->Security->Tokenization and Runtime Manager->Tokenization Service.

Reply:
You don't even see Security unless you have the Anypoint Security - Edge entitlement for your Anypoint Platform account, which is why I said you need that. You have to have that entitlement..